build/webpack-watch-factory.js

```diff
@@ -5,9 +5,12 @@
   This is different than development mode.
 */
 
-const webpack = require('webpack');
-const handleWebpackErrors = require('./webpack/lib/handleWebpackErrors');
+const gulp = require('gulp');
+const watch = require('gulp-watch');
+const { source } = require('./lib/path-helpers');
 
 module.exports = (config) => () => {
-  webpack(config).watch({ }, handleWebpackErrors);
+	watch([ source('**/**') ], () => {
```

Comment:
where does the config go here? is there something further back in the process that is now obsolete?

Reply:
Ah, good catch!
This is no longer needed because within watch now we start webpack\:watch\:build that already uses this productionCiConfig as part of it's build sequence.

Removed.